features/partners.feature

```diff
@@ -4,128 +4,231 @@ Functionaliteit: Huwelijken en geregistreerd partnerschappen van een persoon raa
+      Dan heeft de persoon met burgerservicenummer '999992934' alleen 'partners' met de volgende gegevens
       | burgerservicenummer |
-      | 999992935           |
-      En heeft de partner met burgerservicenummer '999992935' de volgende 'naam' gegevens
-      | naam      | waarde |
-      | voornamen | Merel  |
-
-    Scenario: Actueel partnerschap
-      Gegeven het systeem heeft een persoon met de volgende gegevens
+      | 555550001           |
+      En heeft de partner met burgerservicenummer '555550001' de volgende 'geslachtsaanduiding' gegevens
+      | naam         | waarde |
+      | code         | M      |
+      | omschrijving | man    |
+      En heeft de partner met burgerservicenummer '555550001' de volgende 'naam' gegevens
+      | naam                                 | waarde    |
+      | voornamen                            | Jan       |
+      | adellijkeTitelPredicaat.code         | JH        |
+      | adellijkeTitelPredicaat.omschrijving | jonkheer  |
+      | adellijkeTitelPredicaat.soort        | predicaat |
+      | voorvoegsel                          | van       |
+      | geslachtsnaam                        | Aedel     |
+      En heeft de partner met burgerservicenummer '555550001' de volgende 'geboorte' gegevens
       | naam                | waarde    |
-      | burgerservicenummer | 999993380 |
-      En de persoon heeft een partner met de volgende gegevens
+      | datum               | 19820526  |
+      | plaats.code         | 0599      |
+      | plaats.omschrijving | Rotterdam |
+      | land.code           | 6030      |
+      | land.omschrijving   | Nederland |
+      En heeft de partner met burgerservicenummer '555550001' de volgende 'soortVerbintenis' gegevens
+      | naam         | waarde   |
+      | code         | H        |
+      | omschrijving | huwelijk |
+      En heeft de partner met burgerservicenummer '555550001' de volgende 'aangaanHuwelijkPartnerschap' gegevens
       | naam                | waarde    |
-      | burgerservicenummer | 999992935 |
-      En de partner heeft de volgende naam gegevens
-      | naam              | waarde   |
-      | voornamen (02.10) | Reindert |
-      En de partner heeft de volgende aangaanHuwelijkPartnerschap gegevens
-      | naam                      | waarde   |
-      | datum aangaan (06.10)     | 20040506 |
-      | soort verbintenis (15.10) | P        |
-      En de partner heeft GEEN ontbindingHuwelijkPartnerschap gegevens
+      | datum               | 20171103  |
+      | plaats.code         | 0513      |
+      | plaats.omschrijving | Gouda     |
+      | land.code           | 6030      |
+      | land.omschrijving   | Nederland |
+      En de partner met burgerservicenummer '555550001' heeft GEEN ontbindingHuwelijkPartnerschap gegevens
+
+    @gba
+    Scenario: een actuele partner en een ontbonden partner
+      Gegeven de persoon met burgerservicenummer 999992934 heeft de volgende huwelijken/partnerschappen in de registratie
```

Comment:
Zie eerdere opmerking over waardetabellen in gegeven. Geldt voor meerdere scenario's